### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewProblems_ScreenReader: VoiceOver does not read the shortcut commands of menu items properly.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52454]
**Environment Details:** 
VSCode Version : 1.24.1
Mac OS Version:HighSierra10.13.5 

**Additional Details:**
MAS Violated: MAS1.3.1

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2. Go to View menu and Select Problems.

**Actual:**
VoiceOver does not read the shortcut commands of menu items properly. To navigate to Problems tab the shortcut key is Shift+Command+M but Voice Over only reads Shift+M.

**Expected:**
VoiceOver should read the whole shortcut command.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Problems VoiceOver shortcut commands.pptx](https://github.com/Microsoft/vscode/files/2118979/A11y.VSCode.Problems.VoiceOver.shortcut.commands.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Invalid. In your recording you can see that the voice over properly reads `shift + cmd + m`, which is correct

![screen shot 2018-07-05 at 09 39 33](https://user-images.githubusercontent.com/1926584/42308890-56821296-8037-11e8-8fc4-2107093e28f7.png)
